src/scope.h

```diff
@@ -113,7 +113,9 @@ struct Scope
     int explicitProtection;     // set if in an explicit protection attribute
 
     StorageClass stc;           // storage class
-    char *depmsg;               // customized deprecation message
+
+    // customized deprecation message
+    DeprecatedDeclaration *depdecl;
```

Comment:
You are **changing** the existing style. This is not going to happen. (You're right it's a personal preference - and there are 123 contributors each with their own preferences.)